components/access-management/kfam/bindings.go

```diff
@@ -32,6 +32,15 @@ const ServiceRoleBinding = "servicerolebindings"
 const SERVICEROLEISTIO = "ns-access-istio"
 const USER = "user"
 const ROLE = "role"
+//roleBindingNameMap maps frontend role names to k8s role names and vice-versa
```

Comment:
Why does the KFAM API need to know anything about the frontend?
It seems to me that they should be cleanly separated.
Can you explain why this is needed?

Reply:
@yanniszark the frontend now sends the values 'admin', 'edit' and 'view' to the back-end to identify roles. This happens to be the same names as the default Kubernetes cluster roles and they were passed down as is to the ClusterRole's creation code.

Reply:
@krishnadurai can you add a todo to fix this?

Reply:
I've replied on @yanniszark's issue regarding this: https://github.com/kubeflow/kubeflow/issues/4217#issuecomment-537438075